### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by MJujh**
"Added my information on to my card" Thank for being a helpful project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,6 +122,32 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Your_Name* card START ________  -->
+        <div class="card">
+          <p class="name">Le Chi Bao</p>
+          <p class="contact">
+            
+            <p >Your handle</p>
```
Comment (theAravinthM) [inline index.html:130]: Since these lines make no sense & does not rely on the template, kindly remove these lines.

**Review by theAravinthM — CHANGES_REQUESTED** [review]
Hi @MJujh 
Your PR is very close to getting merged :-) Everything seems good except two lines. The PR needs corrections on 2 lines. Make those corrections & commit, so that I can merge your PR. I have also provided you the comments on the lines to guide to that.

**Comment by MJujh** [comment]
Sorry the delayed respond i have remove the not needed line thanks for pointing the issue out to me

**Comment by theAravinthM** [comment]
Hey great that you made the modifications & your PR is ready to get merged 🎉
Congratulations on your PR Merge, and Thanks for your Contribution! Your card will be live on our Website in a moment!

Now you can follow the Next Steps. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**